test/cypress/cypress/integration/step-definitions/reporting/the-user-agent-report-is-displayed-in-the-table-when.js

```diff
@@ -0,0 +1,16 @@
+import { Then } from 'cypress-cucumber-preprocessor/steps';
```

Comment:
the name of the file "the-user-agent-report-is-displayed-in-the-table-when.js" 
should be 
 "the-user-agent-report-is-displayed-in-the-table-then.js"

Reply:
I'll take it

Reply:
Done !